server/views/pages/curated-lists.njk

```diff
@@ -1,5 +1,9 @@
 {% extends 'layout/default.njk' %}
 
+{% block pageMeta %}
+  <meta name="description" content="Exploration for the incurably curious" />
```

Comment:
@jennpb Would we be able to get something written for this?
As per the ticket, it might be good to use some sort of Keyword / SERP checker.